### ISSUE TITLE
[Errata] Chapter 2, Exam Essentials Typo

### ISSUE THREAD

**Issue by alexroussel** [awsccp/awsccp.github.io#51]
**Please describe the error you're reporting**
It seems there is a typo (SSH --> SSD)

**Please indicate the chapter or appendix, section, and page number where the error appears (if applicable)**
Chapter 2, Exam Essentials, 1st paragraph
The Free Tier lets you run light services such as the t2.micro EC2 instance type and a 30 GB SSH (_-->SSD_) EBS volume.

**Comment by dbclinton** [comment]
Yup. That looks like a typo. I'll make a note for the next edition.
Thanks!
